CHANGELOG.md

```diff
@@ -7,6 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 <!-- next-header -->
 
 ## [Unreleased] - ReleaseDate
+### Changed
+- MSRV is now 1.70 due to changes in `hyper-rustls` (see [v0.27.3](https://github.com/rustls/hyper-rustls/releases/tag/v%2F0.27.3)).
+- client: `hyper-rustls` now uses pure Rust `ring` backend by default with an option to use `aws-lc-rs` through `rustls-tls-aws` feature. ([#140])
+- client: TLS features priority is now `native-tls` > `rustls-tls-aws` > `rustls-tls` ([#141]).
```

Comment:
This is a breaking change and should be prepended with `**BREAKING**`.

Previously, `rustls-tls` was used if all features were enabled.

Also, let's move it to the start of the section (because it's a breaking change)